src/main/features/claim/routes/total.ts

```diff
@@ -19,6 +18,5 @@ export default express.Router()
       .catch(next)
   })
   .post(Paths.totalPage.uri, (req: express.Request, res: express.Response) => {
-    ClaimValidator.claimAmount(req.body.totalAmount)
```

Comment:
There is the following on the `total.njk`. We should remove it since no longer in use.

```
<input type="hidden" name="totalAmount" value="{{ interestTotal.totalAmount }}">
```

Reply:
Deleted it.